i18n.php

```diff
@@ -214,4 +214,21 @@
         'undefined_count'         => '{1,21,31} %1% не определен|]1,Inf] %1% не определено',
         'skipped_count'           => '{1,21,31} %1% пропущен|]1,Inf] %1% пропущено',
     ),
+    'bg'    => array(
+        'snippet_context_choice'  => 'В сет <snippet_undefined><snippet_keyword>%1%</snippet_keyword> има недекларирани стъпки. Изберете в кой Context да бъдат създадени:</snippet_undefined>',
+        'snippet_proposal_title'  => '<snippet_undefined><snippet_keyword>%1%</snippet_keyword> има липсващи стъпки. Можете да ги създадете чрез:</snippet_undefined>',
+        'snippet_missing_title'   => '<snippet_undefined>Използвайте този снипет <snippet_keyword>--snippets-for</snippet_keyword> за да генерирате кода за следните стъпки <snippet_keyword>%1%</snippet_keyword> през конзолата:</snippet_undefined>',
```

Comment:
```suggestion
        'snippet_missing_title'   => '<snippet_undefined>Използвайте този снипет <snippet_keyword>--snippets-for</snippet_keyword>, за да генерирате кода за стъпки <snippet_keyword>%1%</snippet_keyword> през конзолата:</snippet_undefined>',
```

Reply:
What do you think of this @toni-kolev?

Reply:
@martin-georgiev please open a PR if that's a better translation, I'll merge it as is (not perfect Bulgarian support > no Bulgarian support).